lib/optimizely.rb

```diff
@@ -142,7 +142,7 @@ def activate(experiment_key, user_id, attributes = nil)
       experiment = config.get_experiment_from_key(experiment_key)
       send_impression(
         config, experiment, variation_key, '', experiment_key,
-        Optimizely::DecisionService::DECISION_SOURCES['EXPERIMENT'], user_id, attributes
+        Optimizely::DecisionService::DECISION_SOURCES['EXPERIMENT'], user_id, true, attributes
```

Comment:
can we add this parameter after experiment key ? I think adding that between user_id and attributes is a wrong place